src/main/java/de/jeter/chatex/utils/Utils.java

```diff
@@ -69,6 +69,10 @@ public static String replacePlayerPlaceholders(Player player, String format) {
             result = result.replace("%afk", "");
         }
 
+        if (HookManager.checkPurpur() && Config.AFK_PLACEHOLDER.getBoolean()) {
+            result = result.replace("%afk", "");
```

Comment:
That's dupe code here, please include that if to the upper if which checks for essentials already